web/src/main/webapp/script.js

```diff
@@ -12,89 +12,81 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+function searchMe() {
+  var input, filter, stockList, stockListItem, item, txtValue;
+  input = document.getElementById("myInput");
+  filter = input.value.toUpperCase();
+  stockList = document.getElementById("stock-list");
+  stockListItem = stockList.getElementsByTagName("li"); 
+   for (var i = 0; i < stockListItem.length; i++) {
+    item = stockListItem[i];
+    txtValue = item.textContent || item.innerText;
+    if (txtValue.toUpperCase().indexOf(filter) > -1) {
+      stockListItem[i].style.display = "";
+    } else {
+      stockListItem[i].style.display = "none";
+    }
+  }
+}
+
+function loadStocks() {
+  fetch('/list-stock').then(response => response.json()).then((stocks) => {
+    const stockListElement = document.getElementById('stock-list');
+    stocks.forEach((stock) => {
+      stockListElement.appendChild(createStockElement(stock));
+    })
+  });
+}
+
+/** Creates an element that represents a stock */
+function createStockElement(stock) {
+  const stockElement = document.createElement('list');
+  stockElement.className = 'task';
+
+  const titleElement = document.createElement('span');
+  titleElement.innerText = stock.ticker;
+  const priceElement = document.createElement('span');
+  priceElement.innerText = '$' + stock.price;
+
+  stockElement.appendChild(titleElement);
+  stockElement.appendChild(priceElement);
+  return stockElement;
+}
+
 google.charts.load('current', { packages: ['corechart', 'line'] });
 google.charts.setOnLoadCallback(drawChart);
 
 /** Creates a chart and adds it to the page. */
 function drawChart() {
+    
+// Feeds graph random data
+    function myRand(to){
+    var x = Math.floor(Math.random() * to+1);
+    return x;
+}
+
+var my2d = [];
+for(var i = 0; i < 70; i++){
```

Comment:
formatting looks off in this area. Can you run it through prettier.io https://github.com/bchendev/Spring2021-Team11/wiki/Code-Formatting